Datadog.Trace.sln

```diff
@@ -1411,8 +1411,6 @@ Global
 	EndGlobalSection
 	GlobalSection(SharedMSBuildProjectFiles) = preSolution
 		tracer\test\test-applications\Samples.Shared\Samples.Shared.projitems*{3c6dd42e-9214-4747-92ba-78de29aace59}*SharedItemsImports = 4
-		tracer\test\test-applications\Samples.Shared\Samples.Shared.projitems*{472dba92-4fea-4b9a-ba70-0e97b942e12d}*SharedItemsImports = 5
```

Comment:
These two are actually my mistake as I forgot to commit them in one of my earlier PRs. 😃 

I think it's fine to remove them here, but I think they are expected. I do have an active PR (I think) that commits them and a few other changes correctly. 
I'd imagine that they'd be re-added each time VS does a rebuild though - not too important